apps/api/src/hyp3_api/auth.py

```diff
@@ -4,7 +4,7 @@
 import jwt
 
 
-def decode_token(token, required_scopes):
+def decode_token(token):
     try:
         payload = jwt.decode(token, environ['AUTH_PUBLIC_KEY'], algorithms=environ['AUTH_ALGORITHM'])
         return {
```

Comment:
If we're no longer tied to the connexion implementation, I think we can simply `return payload` or `return payload['urs-user-id']` here